types/src/lib.rs

```diff
@@ -93,6 +93,6 @@ pub use types::{
 
 //---------------------------------------------------------------------------------------------------- Feature-gated
 #[cfg(feature = "service")]
-pub mod service;
+pub mod blockchain;
```

Comment:
Ah... the feature name and surrounding docs as well, I think it's just `lib.rs` that references it.